ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -75,7 +75,7 @@ const getIgvTracks = (igvSampleIndividuals, sortedIndividuals, sampleTypes) => {
           if (sample.filePath.endsWith('.cram')) {
             Object.assign(track, {
               format: 'cram',
-              indexURL: `${track.url}.crai`,
+              indexURL: sample.indexFilePath ? igvUrl(sample, 'indexFilePath') : `${track.url}.crai`,
```

Comment:
will you need to update the `indexURL` of the other track types too, or are crams the only ones whose index file locations are changing?

Reply:
good question!

Reply:
confirmed for crams only